<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsInterpolatedLineRendererWidgetBase</class>
 <widget class="QWidget" name="QgsInterpolatedLineRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Second value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsFieldExpressionWidget" name="mFirstValueFieldExpressionWidget" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="QgsFieldExpressionWidget" name="mSecondValueFieldExpressionWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Strock width</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3" rowspan="3">
       <widget class="QgsMeshVariableStrokeWidthButton" name="mPushButtonVariableWidth">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="mRadioButtonVariableWidth">
        <property name="text">
         <string>Variable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="3">
       <widget class="QDoubleSpinBox" name="mSpinBoxWidth"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="mRadioButtonFixWidth">
        <property name="text">
         <string>Fixed</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4" rowspan="3">
       <widget class="QgsUnitSelectionWidget" name="mWidthUnitSelection" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Color</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mLineEditColorMin"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mLineEditColorMax"/>
        </item>
        <item>
         <widget class="QPushButton" name="mButtonLoadColorMinMax">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QgsColorRampShaderWidget" name="mColorRampShaderWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgsfieldexpressionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorRampShaderWidget</class>
   <extends>QWidget</extends>
   <header>raster/qgscolorrampshaderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsUnitSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsunitselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMeshVariableStrokeWidthButton</class>
   <extends>QPushButton</extends>
   <header>qgsmeshvariablestrokewidthwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
